drivers/serial/uart_gecko.c

```diff
@@ -127,6 +129,8 @@ static void uart_gecko_poll_out(const struct device *dev, unsigned char c)
 {
 	const struct uart_gecko_config *config = dev->config;
 
+	pm_policy_state_lock_get(PM_STATE_STANDBY, PM_ALL_SUBSTATES);
```

Comment:
All of these should be `#ifdef` under `CONFIG_PM`.

Reply:
> All of these should be `#ifdef` under `CONFIG_PM`.

not needed, stubs are provided (see `zephyr/pm/policy.h`)

Reply:
Yes, indeed, that is what we assumed